python/ray/tune/registry.py

```diff
@@ -4,6 +4,7 @@
 
 from types import FunctionType
 
+import cloudpickle
```

Comment:
`cloudpickle` technically isn't a dependency of Ray (e.g., it is not in our `setup.py`), so users may not have it installed.

We have our own cloudpickle which you can import with e.g., `import ray.cloudpickle as pickle`. The point of doing this is that it's very easy to have a cloudpickle version mismatch on different machines, but if we ship our own then we don't run into this problem.

Reply:
Fixed!